modules/core/README.md

```diff
@@ -22,7 +22,7 @@ configuration. They are documented briefly below, and in more detail in their ow
 ## Requirements
 
 - AWS account with an access key and secret for programmatic access
-- [Ansible >= 2.6](https://github.com/ansible/ansible/releases)
```

Comment:
You missed out "You will also need to have Ansible 2.6 installed" -> "You will also need to have Ansible 2.7 installed"

Reply:
OH